firmware/powerboard2/code/app/I2C_componentSpecific.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * I2C_componentSpecific.h
+ *
+ *  Created on: Oct 10, 2020
+ *      Author: Shuhang Tan
+ */
+
+#ifndef I2C_COMPONENTSPECIFIC_H_
+#define I2C_COMPONENTSPECIFIC_H_
+
+#include <stdint.h>
+
+//To be updated
+typedef enum
+{
+    I2C_CHANNEL_VBATT,
+	I2C_CHANNEL_5V,
+	I2C_CHANNEL_12V,
+	I2C_CHANNEL_16V,
+
+	I2C_CHANNEL_COUNT,
+} I2C_channel_E;
+
+//To be updated
+static const uint8_t I2C_channelToAddressMapping[I2C_CHANNEL_COUNT] = {

```

Comment:
Don't declare variables in a header file.

Reply:
Now the send function has a weird error ...